packages/installations/src/testing/fake-generators.ts

```diff
@@ -29,7 +29,7 @@ export function getFakeApp(): FirebaseApp {
   return {
     name: 'appName',
     options: {
-      apiKey: 'apiKey',
+      apiKey: 'AIzaSyabcdefghijklmnopqrstuvwxyz1234567',
```

Comment:
This looks like a legit api key value. Would it be nicer to continue  just use a dummy value for testing purpose? or a least comment on the fakeness of this value everywhere (also making this a constant would be ideal)

Reply:
It's a fake api key if you look closely. yeah, I extracted them into a constant. Thanks for the advice.